build.yaml

```diff
@@ -103,6 +103,22 @@ build:
         python setup.py build_ext --inplace --no-cython
       fi
 
+      echo "Installing simulacron"
```

Comment:
hmm, yeah but that way we would have to update the build.yaml for every new release, this way we would have the latest.
Just saw the `git checkout 0.3.1` which should be removed.